examples/bandgap/bandgap_demo.ipynb

```diff
@@ -25,228 +25,55 @@
-      "Headers: {'Authorization': 'Agl8wGJBzyWQgwWqxKXKd4WBmEmWDkq9vE41kyar1enp6gkYywi8CYg3deNKxeblJrVDVbxEpnvB15IJwgXGVirYmB', 'Transfer': 'AgKBxGJWyaBnWJMbrV9GW8NjEP6G3oy0xYJXbDN5JvD309v5lgfeCJ5JomYOwal1aWkrlNO634XENrC15KqmEHzjjJ', 'FuncX': 'AgNyBa5On1qm07rB8YMz1aXN88e1GbK0zOjeEKelrJryOYNaPqSgCy4zQbdM5g5GWKvE9J8mbDxQj8ijyEg97Updxb', 'Petrel': 'Agl8wGJBzyWQgwWqxKXKd4WBmEmWDkq9vE41kyar1enp6gkYywi8CYg3deNKxeblJrVDVbxEpnvB15IJwgXGVirYmB'}\n",
-      "Crawl URL is : http://xtract-crawler-4.eba-ghixpmdf.us-east-1.elasticbeanstalk.com/crawl\n",
-      "Crawl response: <Response [200]>\n",
-      "Crawl ID: 3ea8da43-b07f-4fe9-ad2d-373a4417385e\n",
-      "<Response [200]>\n",
-      "Crawl Status: {'crawl_id': '3ea8da43-b07f-4fe9-ad2d-373a4417385e', 'crawl_status': 'initializing'}\n",
-      "Sleeping before re-polling...\n",
-      "<Response [200]>\n",
-      "Crawl Status: {'crawl_id': '3ea8da43-b07f-4fe9-ad2d-373a4417385e', 'crawl_status': 'initializing'}\n",
-      "Sleeping before re-polling...\n",
-      "<Response [200]>\n",
-      "Crawl Status: {'crawl_id': '3ea8da43-b07f-4fe9-ad2d-373a4417385e', 'crawl_status': 'initializing'}\n",
-      "Sleeping before re-polling...\n",
-      "<Response [200]>\n",
-      "Crawl Status: {'crawl_id': '3ea8da43-b07f-4fe9-ad2d-373a4417385e', 'crawl_status': 'initializing'}\n",
-      "Sleeping before re-polling...\n",
-      "<Response [200]>\n",
-      "Crawl Status: {'crawl_id': '3ea8da43-b07f-4fe9-ad2d-373a4417385e', 'crawl_status': 'initializing'}\n",
-      "Sleeping before re-polling...\n",
-      "<Response [200]>\n",
-      "Crawl Status: {'bytes_crawled': 0, 'crawl_id': '3ea8da43-b07f-4fe9-ad2d-373a4417385e', 'crawl_status': 'crawling', 'files_crawled': 0, 'groups_crawled': 0}\n",
-      "Sleeping before re-polling...\n",
-      "<Response [200]>\n",
-      "Crawl Status: {'bytes_crawled': 0, 'crawl_id': '3ea8da43-b07f-4fe9-ad2d-373a4417385e', 'crawl_status': 'crawling', 'files_crawled': 0, 'groups_crawled': 0}\n",
-      "Sleeping before re-polling...\n",
-      "<Response [200]>\n",
-      "Crawl Status: {'bytes_crawled': 0, 'crawl_id': '3ea8da43-b07f-4fe9-ad2d-373a4417385e', 'crawl_status': 'crawling', 'files_crawled': 0, 'groups_crawled': 0}\n",
-      "Sleeping before re-polling...\n",
-      "<Response [200]>\n",
-      "Crawl Status: {'bytes_crawled': 247427654, 'crawl_id': '3ea8da43-b07f-4fe9-ad2d-373a4417385e', 'crawl_status': 'complete', 'files_crawled': 1, 'groups_crawled': 8}\n",
-      "Our crawl has succeeded!\n",
-      "Queue is empty! Continuing...\n",
-      "Queue is empty! Continuing...\n",
-      "Done curling.\n",
-      "[{'/foundry/_test_foundry_assorted_computational_bandgap_v1.1/foundry_dataframe.json status': True}]\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "for p in requirements['dependencies']:\n",
-    "    f = f.load(p, globus=False, verbose=True)"
+    "from foundry import Foundry\n",
```

Comment:
this looks potentially different from your other two PRs? maybe just double check before we merge this that this notebook is gonna look the way you want it to?